File: task-03/hello world/ip_op/file.go
package main

import (
	"io/ioutil"
	"log"
)

func main() {
	content, err := ioutil.ReadFile("input.txt")
	if err != nil {
		log.Fatal(err)
	}

	err = ioutil.WriteFile("output.txt", content, 0644)
	if err != nil {
		log.Fatal(err)
	}
}

File: task-03/hello world/ip_op/file.cpp
#include <iostream>
#include <fstream>
#include <string>

int main() {
    std::ifstream input("input.txt");
    std::ofstream output("output.txt");
    std::string content;

    if (input && output) {
        while (getline(input, content)) {
            output << content << std::endl;
        }
    }

    input.close();
    output.close();
    return 0;
}
